docs/source/concepts/service.rst

```diff
@@ -166,36 +166,51 @@ argument added to the service API function. Both the request and response contex
 accessed through the inference context for getting and setting the headers, cookies, and
 status codes.
 
-.. code-block:: python
-
-    @svc.api(
-        input=NumpyNdarray(),
-        output=NumpyNdarray(),
-    )
-    def predict(input_array: np.ndarray, ctx: bentoml.Context) -> np.ndarray:
-        # get request headers
-        request_headers = ctx.request.headers
-
-        result = runner.run(input_array)
+.. tab-set::
+
+    .. tab-item:: HTTP
+       :sync: http
+
+       .. code-block:: python
+
+           @svc.api(input=NumpyNdarray(), output=NumpyNdarray())
+           def predict(input_array: np.ndarray, ctx: bentoml.Context) -> np.ndarray:
+               result = runner.run(input_array)
+
+               # set response headers, cookies, and status code 
+               ctx.response.status_code = 202
```

Comment:
Do you have `status_code` support for grpc? Though we can postpone it for now, most likely we'd want to document how to implement and raise a bentomlexception instead.

Reply:
We can add status_code support for gRPC. User can set it to the value that gRPC follows.

Reply:
Yep, for now let's do that.